plugins/outputs/bigquery/bigquery.go

```diff
@@ -0,0 +1,227 @@
+package bigquery
+
+import (
+	"context"
+	"fmt"
+	"reflect"
+	"sync"
+	"time"
+
+	"cloud.google.com/go/bigquery"
+	"golang.org/x/oauth2/google"
+	"google.golang.org/api/option"
+
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/internal"
+	"github.com/influxdata/telegraf/plugins/outputs"
+)
+
+const (
+	defaultOffSetKey   = "offset-key.json"
+	timeStampFieldName = "timestamp"
+)
+
+var defaultTimeout = internal.Duration{Duration: time.Duration(time.Second * 5)}
```

Comment:
Can you please do:
```suggestion
var defaultTimeout = internal.Duration{Duration: 5 * time.Second}
```
as there is a big PR fixing those superfluous expressions (#8835) and Pawel will not be happy if we add more. ;-)